hardware_integration/include/isobus/hardware_integration/virtual_can_plugin.hpp

```diff
@@ -0,0 +1,85 @@
+//================================================================================================
+/// @file virtual_can_plugin.hpp
+///
+/// @brief An OS and hardware independent virtual CAN interface driver for testing purposes.
+/// Any instance connecting to the same channel and in the same process will be able to communicate.
+/// @author Daan Steenbergen
+///
+/// @copyright 2023 Adrian Del Grosso
+//================================================================================================
+#ifndef SOCKET_CAN_INTERFACE_HPP
```

Comment:
Probably need to change the include guards from `SOCKET_CAN_INTERFACE_HPP` to `VIRTUAL_CAN_PLUGIN_HPP`

Reply:
My bad haha, I changed it